Automattic/vip-go-mu-plugins
# Remove Gutenberg Ramp completely

## Description

Note that this is a **time-sensitive release** ― please do not deploy this Pull-Request on the VIP Go Platform until on February **24th** 2021 and after **consulting** with the Platform Support team. Explanation follows.

We are aiming to remove the Gutenberg Ramp plugin. This is the last step in this process. Previous PRs are #1986 and #1958.

This Pull-Request will remove the Gutenberg Ramp plugin from the `vip-go-mu-plugins` repository, and thus remove the plugin from the VIP Go Platform completely. We have taken steps to remove any invocations to the plugin by submitting Pull-Requests, most of those are done by now, and the rest will be done on February 22nd. In some cases Gutenberg Ramp will live in individual repositories, but that is independent from this Pull-Request.

Note that we can only deploy this change when we have performed other actions, such as auditing for the continued use of Gutenberg Ramp, which is accomplished by #1986.

## Changelog Description

### Plugin Removed: Gutenberg Ramp

We completely removed the `gutenberg-ramp` submodule and also the `gutenberg-ramp.php` file from `vip-go-mu-plugins` as the plugin is deprecated and should no longer be used. We've taken steps to ensure that this action will not be disruptive.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [N/A] This change works and has been tested locally (or has an appropriate fallback).
- [-] This change works and has been tested on a Go sandbox. - Yes, multiple, and with multiple configurations of filters.
- [N/A] This change has relevant unit tests (if applicable).
- [N/A] This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Start a VIP Go sandbox for a site. Make note of what editor is loaded currently for a particular post type, Classic or Gutenberg.
2. Check out the Pull-Request on a local VIP Go sandbox: https://patch-diff.githubusercontent.com/raw/Automattic/vip-go-mu-plugins/pull/1992.diff
3. Observe that the `gutenberg-ramp` plugin is not deployed and same with the `gutenberg-ramp.php` file.
4. Make sure the current site works, both front-page and `wp-admin`.
5. Make sure editor functionality remains same as before.
